conf_testing/rules/test_openhab_interface_links.py

```diff
@@ -0,0 +1,97 @@
+import HABApp
+from HABApp.core.Items import get_all_items
+from HABApp.openhab.definitions.rest import ItemChannelLinkDefinition, LinkNotFoundError
+from HABApp.openhab.items import Thing
+from conf_testing.lib.HABAppTests import ItemWaiter, OpenhabTmpItem, TestBaseRule, get_openhab_test_states, get_openhab_test_types
+
+class TestOpenhabInterfaceLinks(TestBaseRule):
+    astro_sun_thing: str
+    item_name: str
+    channel_uid: str
```

Comment:
this can be moved into ``__init__``

Reply:
I wasn't even aware that my syntax meant a static member. Another thing learned.

Reply:
Yea, that's how I basically learned my whole python knowledge ;-)